docs/governance/policies/change-management-versioning.md

```diff
@@ -114,62 +114,62 @@ Initiated [for each need] when: Final change proposal is approved - although ini
+    * The [TODS Board][board] MAY object to their characterization of the need as urgent.
+    * The [TODS Board][board] MAY make changes to the Urgent Working Group members.
+    * If the [TODS Board][board] fails to respond within 48 hours, the Urgent Working Group MAY proceed with their implicit approval.
+3. **Solution Identification**: The Urgent Working Group, managed by the [TODS Manager][manager], MUST identify a solution that meets the urgent need and document it as a PR to the `main` branch of the TODS Repository – outside the normal release cycle.  
+4. **Decision-Making**: A supermajority (two-thirds) of the Urgent Working Group members MUST approve the proposed change to meet the urgent need. If the decision is deadlocked, the decision is escalated to the [TODS Board][board].
 5. **Implementation**: The rest of the implementation and release cycle mirrors the main change-making process.
 
 ## Versions and Release Management
 
 ### Releases
 
-Each change to normative content in the main branch of the ODS Repository MUST be considered a new release and assigned a version number.
+Each change to normative content in the main branch of the TODS Repository MUST be considered a new release and assigned a version number.
 
 * Each release MUST contain one or more changes to normative content which have been approved through the change-making process.
-* Each release MUST be reviewed and approved in the ODS Repository by 2+ members of the [ODS Board][board]  – or their designees – for * accuracy and consistency with the changes’ intent.
+* Each release MUST be reviewed and approved in the TODS Repository by 2+ members of the [TODS Board][board]  – or their designees – for * accuracy and consistency with the changes’ intent.
 * Each release MUST be tagged with its version number.
 * Each release MUST have documentation available.
 * Each release MUST have an entry in `CHANGELOG.md``.
 * Each release MUST be specified as a release on Github.
-* Each release MUST be noticed to any ODS mailing lists or discussion groups.
+* Each release MUST be noticed to any TODS mailing lists or discussion groups.
 * Each release SHOULD have sample datasets (real or exemplary) which cover the normative changes in the release.
 
 #### Pre-releases
 
-Each approved change to normative content in the develop branch of the ODS Repository MAY be considered a pre-release and assigned a pre-release version number.
+Each approved change to normative content in the develop branch of the TODS Repository MAY be considered a pre-release and assigned a pre-release version number.
 
 Pre-releases MAY be made in order to evaluate and work on incrementally-approved changes.
 
 * Each pre-release MUST contain one or more changes to normative content which have been approved through the change-making * process.
```

Comment:
delete '*' character

Reply:
fixed in [8d13ce4](https://github.com/cal-itp/operational-data-standard/pull/78/commits/8d13ce4d9d275ba789b779e7463a8429564075ba)